examples/ippool/ippool.yaml

```diff
@@ -1,35 +1,40 @@
 ---
-apiVersion: cluster.x-k8s.io/v1alpha3
+apiVersion: cluster.x-k8s.io/v1alpha4
 kind: Cluster
 metadata:
   name: ${CLUSTER_NAME}
+  namespace: ${NAMESPACE}
 spec:
   clusterNetwork:
     services:
       cidrBlocks: ["10.96.0.0/12"]
     pods:
-      cidrBlocks: ["192.168.0.0/16"]
-    serviceDomain: "cluster.local"
+      cidrBlocks: ["192.168.0.0/18"]
   infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
     kind: Metal3Cluster
     name: ${CLUSTER_NAME}
+    namespace: ${NAMESPACE}
   controlPlaneRef:
     kind: KubeadmControlPlane
-    apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+    apiVersion: controlplane.cluster.x-k8s.io/v1alpha4
     name: ${CLUSTER_NAME}-controlplane
+    namespace: ${NAMESPACE}
 ---
 apiVersion: ipam.metal3.io/v1alpha1
 kind: IPPool
 metadata:
   name: pool1
+  namespace: ${NAMESPACE}
 spec:
   clusterName: ${CLUSTER_NAME}
+  namePrefix: ${CLUSTER_NAME}-prov
 ---
 apiVersion: ipam.metal3.io/v1alpha1
 kind: IPClaim
 metadata:
-  name: Claim1
+  name: ${CLUSTER_NAME}-controlplane-template-0-provisioning-pool
```

Comment:
can you maybe update this example name in https://github.com/metal3-io/ip-address-manager/blob/a1958aab9b77fb571c57e83a9306870f2011ea8a/docs/api.md too?

Reply:
Sure, done.

Reply:
Please see #42 I updated the documentation there  since this PR is only for fixing examples generation.